Make.inc

```diff
@@ -362,6 +362,18 @@ export PKG_CONFIG_LIBDIR = $(JULIAHOME)/usr/lib/pkgconfig
 # Figure out OS and architecture
 BUILD_OS := $(shell uname)
 
+# Set ISMSYS_MINGW64 to 0 or 1
+OSNAME := $(shell uname -o | tr '[:lower:]' '[:upper:]')
+ISMSYS_MINGW64 := 0
+ifneq (,$(findstring MSYS,$(OSNAME)))
+TRIPLET := $(shell $(CC) -dumpmachine)
```

Comment:
this is already `ARCH`, so we should not be making a duplicate of this value computation here. Indeed, we already handle all of this later (around line 403), so none of this should be duplicated here.

Reply:
Does the sequence of statements not apply to bash scripts?
Because moving the setting of  ISMSYS_MINGW64 to after it use used seems wrong to me.

Tim S.

Reply:
Order does apply, though this is not bash, and the use seems to be around 636 while the existing definition of this appears to be before that, around 403

Reply:
Yeah. but ARCH is after my use of ISMSYS_MINGW64